dns.tf

```diff
@@ -1,5 +1,9 @@
+locals {
+  dns_subdomain = "pcf"
```

Comment:
Could you have this default to `"${var.env_name}"`? We like to keep the terraforming-* templates in sync with bbl's Terraform templates where we can.